salalib/axialmap.cpp

```diff
@@ -2115,7 +2120,7 @@ bool ShapeGraph::angularstepdepth(Communicator *comm)
    std::string stepdepth_col_text = "Angular Step Depth";
    int stepdepth_col = m_attributes.insertColumn(stepdepth_col_text.c_str());
 
-   int tulip_bins = 1024;
+   size_t tulip_bins = 1024;

```

Comment:
Why not set to `tulip_bins = 513` with a comment explaining this? Probably not a big deal, though, the compiler might do this anyway.